src/java/com/zimbra/qa/importer/ParserAccount.java

```diff
@@ -1,156 +1,158 @@
+		
+
+		
+		// Remove any skipped attributes
+		List<String> skipped = Arrays.asList("cn", "mail", "objectClass", "sn", "uid", "userPassword", "zimbraId", 
+				"zimbraMailHost", "zimbraMailTransport", "zimbraLastLogonTimestamp", "zimbraMailDeliveryAddress",
+				"zimbraZimletAvailableZimlets", "zimbraPrefStandardClientAccessilbityMode");
+		
+		for ( String key : skipped) {
+			if (attrs.containsKey(key))
+				attrs.remove(key);
+		}
+
+		// BEGIN: zimbraMailSieveScript - a multiline attribute
+		RandomAccessFile rafReader = null;		
+		try{
+			rafReader = new RandomAccessFile(InputFile, "r");
+			String line;
+			StringBuilder sb = null;
+			while ( (line = rafReader.readLine()) != null ) {
+				
+				if ( line.startsWith("zimbraMailSieveScript: ") ) {
+					sb = new StringBuilder();
+					sb.append(line.substring("zimbraMailSieveScript: ".length())).append("\n");
+					while ( (line = rafReader.readLine()) != null ) {
+						if ( line.startsWith("zimbra") )
+							break;
+						sb.append(line).append("\n");
+					}
+					break;
+				}
+			}
+			if ( sb != null ) {
+				if ( attrs.containsKey("zimbraMailSieveScript") )
+					attrs.remove("zimbraMailSieveScript");
+				attrs.put("zimbraMailSieveScript", sb.toString());
+			}
+			// END: zimbraMailSieveScript
+			
+			
+			// BEGIN: zimbraQuotaWarnMessage - the default message contains the ":" character, which breaks the ParserBase.parseInputFile() method
+			String zimbraQuotaWarnMessage = null;
+			for ( String key : attrs.keySet()) {
+				if (key.startsWith("zimbraQuotaWarnMessage")) {
+					zimbraQuotaWarnMessage = key;
+					break;
+				}
+			}
+			attrs.remove(zimbraQuotaWarnMessage);
+			
+			rafReader.seek(0);
+			
+			while ( (line = rafReader.readLine()) != null ) {
+				
+				if ( line.startsWith("zimbraQuotaWarnMessage: ") ) {
+					attrs.put("zimbraQuotaWarnMessage", line.substring("zimbraQuotaWarnMessage: ".length()));
+					break;
+				}
+			}		
+			// END: zimbraQuotaWarnMessage
+			
+			// BEGIN: zimbraInterceptSubject - the default message contains the ":" character, which breaks the ParserBase.parseInputFile() method
+			String zimbraInterceptSubject = null;
+			for ( String key : attrs.keySet()) {
+				if (key.startsWith("zimbraInterceptSubject")) {
+					zimbraInterceptSubject = key;
+					break;
+				}
+			}
+			attrs.remove(zimbraInterceptSubject);
+			
+			rafReader.seek(0);
+			while ( (line = rafReader.readLine()) != null ) {
+				
+				if ( line.startsWith("zimbraInterceptSubject: ") ) {
+					attrs.put("zimbraInterceptSubject", line.substring("zimbraInterceptSubject: ".length()));
+					break;
+				}
+			}		
+			// END: zimbraInterceptSubject
+			
+			// BEGIN: zimbraNewMailNotificationBody - the default message contains the ":" character, which breaks the ParserBase.parseInputFile() method
+			String zimbraNewMailNotificationBody = null;
+			for ( String key : attrs.keySet()) {
+				if (key.startsWith("zimbraNewMailNotificationBody")) {

```

Comment:
Create a constant for zimbraNewMailNotificationBody

Reply:
there are lots of Strings in the project and this class as well.
will the need to create for all,
we can do this in another ticket to maintain uniformity of code.